packages/cli/src/helpers/create-consumable-html.ts

```diff
@@ -138,12 +139,18 @@ export function bucketPhaseValues(
   return buckets;
 }
 
+export interface ParsedTitleConfigs {
+  servers: Array<{ name: string }>,
+  plotTitle: string | undefined
+}
+
 /**
  * Override the default server and plot title attributes
  *
- * @param tbConfig - Concerned only about the "servers" and "plotTitle" attribute
+ * @param tbConfig - Concerned only about the "servers" and "plotTitle"
+ *   attribute
  */
-export function resolveTitles(tbConfig: Partial<ITBConfig>) {
+export function resolveTitles(tbConfig: Partial<ITBConfig>): ParsedTitleConfigs {
   const reportTitles = {
```

Comment:
lets specify the type for `reportTitles: ParsedTitleConfigs`

Reply:
and then remove the casting `as any`

Reply:
done!

Reply:
I removed the trailing commas in the code